services/cargohold/src/CargoHold/S3.hs

```diff
@@ -195,10 +195,14 @@ mkKey :: V3.AssetKey -> S3AssetKey
 mkKey (V3.AssetKeyV3 i r) = S3AssetKey $ "v3/" <> retention <> "/" <> key
   where
     key = UUID.toText (toUUID i)
+    -- This is the prefix used to store on S3, which currently is in sync
+    -- with the JSON representation but does not necessarily need to be
```

Comment:
It would make it easier if it had to be, though, no?  I would require the two to always match and use a single function to create the strings.

Reply:
Yeah, was about to do so :)

Reply:
https://github.com/wireapp/wire-server/pull/252/commits/7a9f01f98420e6224c266dbe4687300c699dd5d4